wazo_call_logd/cel_interpretor.py

```diff
@@ -292,6 +292,67 @@ def interpret_wazo_user_missed_call(self, cel, call):
         call.source_line_identity = identity_from_channel(cel.channame)
         return call
 
+    def interpret_wazo_call_log_destination(self, cel, call):
+        extra = extract_cel_extra(cel.extra)
+        if not extra:
+            return
+
+        extra_tokens = extra['extra'].split(',')
+        extra_dict = dict()
+        for token in extra_tokens:
+            key = token.split(': ')[0].strip()
+            value = token.split(': ')[1].strip()
+            extra_dict[key] = value
+
+        if 'type' not in extra_dict.keys():
+            raise KeyError('required destination type is not found.')
```

Comment:
This will cause the processing of the call to crash, right? This will cause the call log to be absent altogether from the API. This is too harsh for that kind of error. We should instead log an error and treat the call log destination as empty.

Reply:
okay, I will change it.